apps/demos/utils/visual-tests/matrix-test-helper.js

```diff
@@ -54,6 +69,33 @@ export function getChangedFiles() {
     && JSON.parse(readFileSync(changedFilesPath));
 }
 
+export function globalReadFrom(basePath, relativePath, mapCallback) {
+  const absolute = join(basePath, relativePath);
+  if (existsSync(absolute)) {
+    const result = readFileSync(absolute, 'utf8');
+    return (mapCallback && result && mapCallback(result)) || result;
+  }
+  return null;
+}
+
+export function changeTheme(dirName, relativePath, demoPath, theme) {
```

Comment:
3 starting params looks like overkill. As I see you always use together at least 1st and 2nd. Why dont change it with, say,  (demosRoot, demoPath, theme) and join some paths in a callers.

Reply:
https://github.com/DevExpress/DevExtreme/pull/26748/commits/5099ddceea7e349f16bf267aaa3e4127e5ef32bf